### ISSUE TITLE
Help me find things that sound off / badly tuned

### ISSUE THREAD

**Issue by inversephase** [kieranhj/pop-beeb#184]
This especially applies to the bassy bits. I'll just edit the source and re-render VGMs for a less buggy conversion experience.

**Comment by inversephase** [comment]
I'll start. story1 needs fixing for sure

**Comment by inversephase** [comment]
oh dear. source for story1 was overwritten, that one needs to just be fixed in conversion =\

**Comment by kieranhj** [comment]
The exiting level 4 tune sounded a bit wonky to me?

**Comment by kieranhj** [comment]
The potion ditties sound different than before - they don't seem to finish properly?  (Or have channels missing.)

**Comment by inversephase** [comment]
can you tell me if the new potion ditties in slack exhibit the same behavior?

**Comment by simondotm** [comment]
They do. I seem to have broken something in the script with the quad-tone "fix" I put in. Investigating...

**Comment by simondotm** [comment]
Yeah, if I turn off that "fix", it sounds fine. But then of course, the tuning issues are back. Hmmm...

**Comment by inversephase** [comment]
for the sake of the deadline, I think we should just turn off the fix for potions and leave it on for others... I know the struggle of wanting something that builds perfectly from the source but I think we've got bigger fish to fry

**Comment by simondotm** [comment]
- [ ] m-intro-wrongsignaturevgm (beep at end)
- [ ] m-killjaffar (out of tune ping mid section)
- [ ] m-lifepotion-sliced (volumes broken)
- [ ] m-potion-sliced (volumes broken)
- [ ] m-story2 (beep at end)
- [ ] m-story5-end-merge-bla (tuning off, beep at end)

These are all artefacts when the tunes are processed with the "quad tone fix"

**Comment by simondotm** [comment]
Trying out the new vgms from @inversephase 
m-killjaffar-usethisone : does fix the out of tune bit
m-lifepotion-isitfixed : is different, but not fixed
m-potion-isthisfixed : is again different, but not fixed
m-story2-shouldfixbeep : does indeed fix the beep

I'm going to try reverting the script, and only processing the original ones we found tuning issues with

**Comment by simondotm** [comment]
Ok so without any "quad tone fix" in the conversion, these vgms have tuning issues:

- [ ] m-cutscene-pre2_4_6_C
- [ ] m-cutscene-pre2_4_6_C-alt
- [ ] m-e4-killedbyshadow
- [ ] m-intro-wrongsignaturevgm
- [ ] m-lifepotion-sliced (potion-sliced is ok)
- [ ] m-playerdeath-sword
- [ ] m-playerdeath-reg
- [ ] m-story4
- [ ] m-story4-endsright
- [ ] m-story5-end-merge-bla

Until I can really properly fix the script, the best I can do is find the best combination of script with quad tone fix on/off to minimize the number of problematic tunes.

**Comment by simondotm** [comment]
Ok I have implemented an additional patch to the converter which has largely fixed all of the rogue volume issues. 
Problem was that it wasn't coping with situations where channel 3 volume might be keyed off AFTER it has detected a non-silent volume setting on channel 2 AND tuned noise is currently playing on channel 3. 
I went through the outputs and they sound much better. There are still issues with lifepotion though so I will raise a separate issue for that.
